ui/css/treeherder-job-buttons.css

```diff
@@ -0,0 +1,608 @@
+.btn-purple-classified[disabled]:focus,
+.btn-purple-classified[disabled]:active,
+.btn-purple-classified[disabled].active,
+fieldset[disabled] .btn-purple-classified:hover,
+fieldset[disabled] .btn-purple-classified:focus,
+fieldset[disabled] .btn-purple-classified:active,
+fieldset[disabled] .btn-purple-classified.active {
+  background-color: #9002c2;
+  border-color: #9002c2;
+  color: white;
+}
+
+/* Light blue, coalesced */
+.btn-ltblue,
+.btn-ltblue-count,
+.btn-ltblue-classified,
+.btn-ltblue-classified-count,
+.btn-ltblue-count:hover,
+.btn-ltblue-classified-count:hover {
+  color: #81b8ed;
+  font-weight: bold;
+}
+.btn-ltblue:hover,
+.btn-ltblue-classified:hover,
+.btn-ltblue:focus,
+.btn-ltblue:active,
+.btn-ltblue.active {
+  border-color: #488ae9;
+  background-color: #488ae9;
+  color: white;
+}
+.btn-ltblue.disabled:hover,
+.btn-ltblue.disabled:focus,
+.btn-ltblue.disabled:active,
+.btn-ltblue.disabled.active,
+.btn-ltblue[disabled]:hover,
+.btn-ltblue[disabled]:focus,
+.btn-ltblue[disabled]:active,
+.btn-ltblue[disabled].active,
+fieldset[disabled] .btn-ltblue:hover,
+fieldset[disabled] .btn-ltblue:focus,
+fieldset[disabled] .btn-ltblue:active,
+fieldset[disabled] .btn-ltblue.active {
+  border-color: #81b8ed;
+}
+
+/* Light gray, pending */
+.btn-ltgray,
+.btn-ltgray-count,
+.btn-ltgray-classified,
+.btn-ltgray-classified-count {
+  color: #e0e0e0;
+}
+.btn-ltgray-count:hover,
+.btn-ltgray-classified-count:hover {
+  color: #808080;
+}
+.btn-ltgray:hover,
+.btn-ltgray-classified:hover,
+.btn-ltgray:focus,
+.btn-ltgray:active,
+.btn-ltgray.active {
+  border-color: #808080;
+  color: #808080;
+}
+.btn-ltgray.disabled:hover,
+.btn-ltgray.disabled:focus,
+.btn-ltgray.disabled:active,
+.btn-ltgray.disabled.active,
+.btn-ltgray[disabled]:hover,
+.btn-ltgray[disabled]:focus,
+.btn-ltgray[disabled]:active,
+.btn-ltgray[disabled].active,
+fieldset[disabled] .btn-ltgray:hover,
+fieldset[disabled] .btn-ltgray:focus,
+fieldset[disabled] .btn-ltgray:active,
+fieldset[disabled] .btn-ltgray.active {
+  color: #e0e0e0;
+  border-color: #e0e0e0;
+}
+
+/* Mid gray (presently unused) */
+.btn-mdgray,
+.btn-mdgray-count,
+.btn-mdgray-classified,
+.btn-mdgray-classified-count,
+.btn-mdgray-count:hover,
+.btn-mdgray-classified-count:hover {
+  background-color: #bfbfbf;
+  border-color: #bfbfbf;
+}
+.btn-mdgray:hover,
+.btn-mdgray-classified:hover,
+.btn-mdgray:focus,
+.btn-mdgray:active,
+.btn-mdgray.active {
+  background-color: #b2b2b2;
+  border-color: #a6a6a6;
+}
+.btn-mdgray.disabled:hover,
+.btn-mdgray.disabled:focus,
+.btn-mdgray.disabled:active,
+.btn-mdgray.disabled.active,
+.btn-mdgray[disabled]:hover,
+.btn-mdgray[disabled]:focus,
+.btn-mdgray[disabled]:active,
+.btn-mdgray[disabled].active,
+fieldset[disabled] .btn-mdgray:hover,
+fieldset[disabled] .btn-mdgray:focus,
+fieldset[disabled] .btn-mdgray:active,
+fieldset[disabled] .btn-mdgray.active {
+  background-color: #bfbfbf;
+  border-color: #bfbfbf;
+}
```

Comment:
This `.btn-mdgray` was created Nov 13' in https://github.com/mozilla/treeherder/commit/88942de2a093bedc1a857a35449854463b2a3215 and is currently unused. It does indeed seem a nice mid-point between our pending and running, so I left it in and marked it as such. I could remove it though if we want.